modules/OFConnectionManager2/module/src/bundle.c

```diff
@@ -541,18 +541,19 @@ bundle_task(void *cookie)
                     /* rv is OK, ERROR, or CONTINUE, then allow next message.
                      * The obj should have been duplicated in the OFStateManager
                      * or driver. It is safe to free the obj.
-                     * TODO: special case if this obj is the last message in
-                     *       subbundle and rv is CONTINUE, we still treat it as
-                     *       PENDING case.
+                     * TODO: Due to driver/OFStateManager doesn't know whether
+                     *       it can start to proccess the accumulated msgs,
```

Comment:
i think subbundle_finish provides a hook by which whoever is interested in knowing when all the messages in a  subbundle have been processed.  (we also have the corresponding subbundle_start.)  does subbundle_finish not handle this use case?

Reply:
Remove TODO